app/src/js/reducers/api.js

```diff
@@ -1,22 +1,21 @@
 'use strict';
-import { set } from 'object-path';
-import { get as getToken, set as setToken } from '../utils/auth';
 
+import { get as getToken, set as setToken } from '../utils/auth';
+import { createReducer } from '@reduxjs/toolkit';
 import {
   DELETE_TOKEN,
   LOGIN,
-  LOGIN_INFLIGHT,
   LOGIN_ERROR,
+  LOGIN_INFLIGHT,
   LOGOUT,
   REFRESH_TOKEN,
   REFRESH_TOKEN_ERROR,
   REFRESH_TOKEN_INFLIGHT,
   SET_TOKEN
 } from '../actions/types';
-import { createReducer } from '@reduxjs/toolkit';
 
 export const initialState = {
-  authenticated: !!getToken(),
+  authenticated: getToken() != null,
```

Comment:
Can you use !== null here?

Reply:
I specifically chose `!=` to catch `undefined` as well as `null`, to be safe, but I'm not strongly opposed to changing it to `!==`, if you would prefer it.

Reply:
I have pretty strong feelings about `==` and `!=` particularly when comparing 0 '' undefined null false and true (see p 121 Javascript the good parts.)

Reply:
Yep, fair enough.